apps/web/src/config/constants/tokenLists/pancake-supported-onramp-currency-list.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "PancakeSwap OnRamp List",
```

Comment:
can we make this file to `https://github.com/pancakeswap/token-list`

Reply:
yeah i created a pr to the token list repo earlier https://github.com/pancakeswap/token-list/pull/188